tests/lib/rules/no-trailing-spaces.js

```diff
@@ -67,6 +67,26 @@ ruleTester.run("no-trailing-spaces", rule, {
     invalid: [
         {
             code:
+            "var short2 = true;\n" +
```

Comment:
Can you add a test case with shorthand properties and methods to match the issue?